src/components/AddBookForm.js

```diff
@@ -37,27 +50,34 @@ export const AddBookForm = () => {
   }, [message]);
 
   return (
-    <div>
-      <h2>ADD NEW BOOK</h2>
-      <p>{message}</p>
-
-      <form method="POST" onSubmit={handleFormSubmit}>
-        <input 
-          type="text"
-          placeholder="Author Name" 
-          value={author}
-          name="author"
-          onChange={(e) => setAuthor(e.target.value)}
-        />
-        <br/>
+    <div className="border mx-20 p-10 mt-10">
```

Comment:
Your form is not the same as in the Zeplin.

- [ ] Please kindly check Zeplin design and make sure it is the same;

Your design:
<img width="1440" alt="Screen Shot 2021-11-06 at 01 39 20" src="https://user-images.githubusercontent.com/60043698/140575714-0f38d6cc-1bca-439a-8198-f6434280d820.png">

Zeplin's:
<img width="1440" alt="Screen Shot 2021-11-06 at 01 39 37" src="https://user-images.githubusercontent.com/60043698/140575760-97705943-9532-4f17-8fb0-e363bc049d8e.png">

Reply:
Thanks I have updated the design as suggested.